### PR TITLE
Added Event to function

### PR DESCRIPTION
**Pull request by MrSahilShaikh**
Fix the issue #56
Added Event to function that change the blockchain state or do write operation on blockchain.
@J0SAL review changes.

### CODE REVIEW

**Comment by J0SAL** [comment]
@gurjeetsinghvirdee can you review this

File: backend/contracts/ExpenseTracker.sol [hunk 1]
```diff
@@ -45,8 +46,10 @@ contract ExpenseTracker {
         tracker[msg.sender].total += 1;
         tracker[msg.sender].transactions.push(_id);
         all_ids[_id] = true;
+        emit addexpense(_id,_amount, _description, _category, _date);
     }
 
+    event addincome(string id,uint amount,string description,string category,string date);
```
Comment (J0SAL) [inline backend/contracts/ExpenseTracker.sol:52]: please move all the event declarations at the top

**Review by J0SAL — CHANGES_REQUESTED** [review]

File: backend/contracts/ExpenseTracker.sol (hunk 1 above)
Comment (MrSahilShaikh) [reply backend/contracts/ExpenseTracker.sol:52]: ok

**Comment by MrSahilShaikh** [comment]
@J0SAL I did changes. please review it

**Pull request merged**